frontend/packages/schema-editor/src/components/SchemaInspector/ItemFieldsTab/ItemFieldsTable/ItemFieldsTableRow.tsx

```diff
@@ -46,6 +54,13 @@ export const ItemFieldsTableRow = ({
     );
   };
 
+  const typeLabel =
+    isField(fieldNode) && typeOptions.find(({ value }) => value === fieldNode.fieldType)?.label;
+
+  const kindNames = useKindOptions();
+  const notReferenceKind = fieldNode.objectKind !== ObjectKind.Reference;
```

Comment:
siden at vi returnerer reference knapp så måtte jeg å utelukke `reference `valget

Reply:
Denne sjekken kan du flytte til den nye komponenten du har laget. Den er jo irrelevant for resten av `ItemFieldsTableRowProps`. Egentlig holder det kanskje at du sender inn `fieldNode` som en prop til den nye komponenten for så å utlede `typeLabel` og `kindLabel` der inne?

Reply:
Flyttet med `typeLable `og `kindLable`